fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1129,6 +1129,7 @@ ReadYourWritesTransaction::ReadYourWritesTransaction(Database const& cx)
     options(tr), deferredError(cx->deferredError) {
 	std::copy(cx.getTransactionDefaults().begin(), cx.getTransactionDefaults().end(),
 	          std::back_inserter(persistentOptions));
+	debugTransaction( deterministicRandom()->randomUniqueID() );
```

Comment:
Will it provide more flexibility if we can add a knob to toggle` debugTransaction`?

Reply:
I don’t know enough of the context here, but traditionally something like this would probably be enabled by an option rather than a knob.

Reply:
Sorry, I didn't mean to create a regular PR for this. This is purely experimental to evaluate the overhead. But I agree with AJ, it will be enabled by an option in future.